etna/images/migrations/0002_migrate_old_image_data_to_new_model.py

```diff
@@ -0,0 +1,74 @@
+# Generated by Django 3.2.14 on 2022-09-20 15:53
+
+from django.db import migrations
+
+
+def migrate_forwards(apps, schema_editor):
+    # Copy images to new table, retaining IDs and all other values
+    Image = apps.get_model("wagtailimages", "Image")
+    CustomImage = apps.get_model("images", "CustomImage")
+    to_create = []
+    for item in Image.objects.all().values().iterator():
```

Comment:
See above: I'm really only concerned about the number of objects we have in memory at any one time. If I only send 1000 items to `bulk_create`, setting `batch_size` won't affect anything.

Reply:
interesting, I would have that setting the iterator to that batch_size would load that much into memory but I guess they're not loaded until accesed. My understanding is that at that level it would load that much into memory.

Reply:
It's size of the `to_create` list that we're limiting.